tests/tui/conftest.py

```diff
@@ -0,0 +1,131 @@
+# -*- coding: utf-8 -*-
+# conftest.py
+import os
+import shutil
+from pathlib import Path
+
+import pytest
+
+from halfpipe import resource
+from halfpipe.tui.base import MainApp  # Ensure path aligns with your project structure
+
+from .create_mock_bids_dataset import create_bids_data
+
+
+# Custom fixture that returns a specific path, this is needed so that the path in the snapshot is always the same
+# If the path was variable then the snapshot would yield failure.
+@pytest.fixture(scope="session")
+def fixed_tmp_path() -> Path:
+    path = Path("/tmp/tui_test/")
+    path.mkdir(parents=True, exist_ok=True)  # Ensure the path exists
+    return path
+    # Clean up after the test, is this needed?
+    # shutil.rmtree(path)
+
+
+# Define the fixture with module scope, one subject, three tasks
+@pytest.fixture(scope="session")
+def downloaded_data_path(fixed_tmp_path) -> Path:
+    # dataset = Dataset(
+    #     name="PIOP1",
+    #     openneuro_id="ds002785",
+    #     openneuro_url="https://openneuro.org/datasets/ds002785/versions/2.0.0",
+    #     url="https://github.com/OpenNeuroDatasets/ds002785.git",
+    #     paths=[
+    #         "sub-0001/anat",
+    #         "sub-0001/func/sub-0001_task-anticipation_acq*",
+    #         "sub-0001/func/sub-0001_task-workingmemory_acq*",
+    #         "sub-0001/func/sub-0001_task-restingstate_acq*",
+    #     ],
+    # )
+    #
+    # data_path = fixed_tmp_path / "ds002785/"
+    # dataset.download(data_path)
+    tasks_conditions_dict = {
+        "anticipation_acq-seq": ["cue_negative", "cue_neutral", "img_negative", "img_neutral"],
+        "workingmemory_acq-seq": ["active_change", "active_nochange", "passive"],
+        "restingstate_acq-mb3": [],
+    }
+    data_path = fixed_tmp_path / "ds002785"
+    create_bids_data(data_path, number_of_subjects=1, tasks_conditions_dict=tasks_conditions_dict)
+    return data_path
+
+
+@pytest.fixture(scope="session")
+def atlases_maps_seed_images_path(fixed_tmp_path) -> Path:
+    # set atlases, seed maps and spatial maps
+    test_online_resources = {
+        "FIND_ica_maps_2009.nii.gz": "https://drive.google.com/file/d/1XnFGm9aCcTIuXgKZ71fDqATBJWAxkInO/view?usp=drive_link",
```

Comment:
Are these links to your personal Google Drive? If so, I would suggest to use OSF or Figshare as an alternative, so that they will be online indefinitely

Reply:
yes, i would like to relocate these to something more public than my google drive